AttributeAndMethodLayOut.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Objects;
+/**
+ *  Class that help to structure the lay out of attribute name and method name
+ *  with their type and visibility.
+ */
+public class AttributeAndMethodLayOut {
+    /**
+     * Attributes and methods has visibility , name and type.
+     *  e.g
+     *    +Id: int
+     *    +managerID()
+     */
+    private String name;
+    private  String type;
+    private String visibility;
+
+    /**
+     * Constructor to initializes attribute and methods with their parameter.
+     * @param name The name of the object.
+     * @param type Type of the object.
+     * @param visibility visibility of the object.
+     */
+    public AttributeAndMethodLayOut(String name, String  type, String visibility){
```

Comment:
Some tests for the validity of the parameters seems necessary to avoid any unwanted inputs.